CoreFoundation/cmake/modules/FindICU.cmake

```diff
@@ -0,0 +1 @@
+../../../cmake/modules/FindICU.cmake
```

Comment:
Ugh, the symlink is painful since it will break checkouts on Windows.  Can you please copy?

Reply:
Fixed